src/components/button/AposButton.vue

```diff
@@ -27,23 +32,39 @@ export default {
       required: true,
       type: String
     },
-    modifiers: Array,
+    modifiers: {
+      type: Array,
+      default() {
+        return [];
+      }
+    },
     disabled: Boolean,
     busy: Boolean,
-    icon: String,
-    type: String,
+    icon: {
+      type: String,
+      default: null
+    },
+    type: {
+      type: String,
+      default: null
+    },
     iconOnly: Boolean,
     iconColor: {
       type: String,
       default: ''
```

Comment:
Should this be `null`? Seems like we're `null`ing all other string defaults here.

Reply:
We're actually no longer using this as of this PR. I switched to `currentColor`. @stuartromanek should the icon ever be a different color from the text? If not we can simply delete this prop.

Reply:
looks like we're still using `iconColor` in `AposPiecesManagerToolbar` ?
It provides us with a way to 'activate' a button that would be dependent on the component's state